update to v13.5.212.10

danielnachun opened the pull request:
I've gotten this version to build locally on both Linux and macOS.  The major changes are as follows:

- One resource `trace_event` is no longer and five new ones `abseil-cpp`, `fast_float`, `fp16`, `highway`, and `partition_allocator`.  No mechanism appears to be provided to devendor these at the moment.
- Two patches `0001-Allow-non-clang-snapshot-toolchain.patch` and `noicf.patch` are no longer needed
- `_LIBCPP_DISABLE_AVAILABILITY` needs to be added to the `CXXFLAGS` but the environment variable is ignored so I had to add it by patching the `defines` variable in several of the `BUILD.gn` files.
- The Clang version on macOS must be explicitly defined as it now hard-codes defaults 
- A new argument `icu_use_data_file=false` must be added when using the system ICU.
- An ugly hack was needed on macOS to avoid using the vendored `libc++`.  I will see if I can convert it to patch once CI is passing.

regro-cf-autotick-bot committed:
update to v13.5.212.10

regro-cf-autotick-bot committed:
MNT: Re-rendered with conda-build 24.11.2, conda-smithy 3.45.0, and conda-forge-pinning 2024.12.14.08.05.48

Commit:
MNT: Re-rendered with conda-build 25.1.1, conda-smithy 3.45.4, and conda-forge-pinning 2025.01.24.17.08.54

danielnachun committed:
fix build for macOS

danielnachun committed:
fix build for linux

danielnachun committed:
fix build for linux

danielnachun committed:
fix build for linux

danielnachun committed:
reword

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ Selectors are suggested to take a ``<two spaces>#<one space>[<expression>]`` form. See lines [62]
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/16681646495](https://github.com/conda-forge/conda-forge-webservices/actions/runs/16681646495). Examine the logs at this URL for more detail.</sub>

danielnachun committed:
MNT: Re-rendered with conda-build 25.5.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.07.31.21.21.45

danielnachun committed:
update compilers

danielnachun committed:
MNT: Re-rendered with conda-build 25.5.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.07.31.21.21.45

danielnachun committed:
update compilers

danielnachun committed:
use lld on macOS

danielnachun committed:
MNT: Re-rendered with conda-build 25.5.0, conda-smithy 3.51.1, and conda-forge-pinning 2025.07.31.21.21.45

danielnachun committed:
more fixes

danielnachun committed:
more fixes

danielnachun committed:
more fixes

danielnachun committed:
more fixes

danielnachun committed:
more fixes

danielnachun committed:
more fixes

danielnachun committed:
more fixes

danielnachun committed:
add patches

danielnachun committed:
update patches

danielnachun commented:
The native builds are passing.  The remaining issues are all related to cross-compilation:
- On `osx-arm64`, `gn` does finish but then we get an error similar to the ones that were happening before the necessary `LDFLAGS` were set to force the linker to use the `conda-forge` `libcxx`.  I believe this is happening because the build system is incorrectly targeting x86 when cross-compiling for arm64, and it ends up rejecting the `conda-forge` `libcxx` because it thinks it is the wrong architecture.
- On `linux-aarch64`, `gn` is failing because it is not picking up the `gcc_toolchain` we have added and this leads to some conflicts with the `clang_toolchain` that it is using instead.  The patch probably needs to be revised.
- On `linux-ppc64le`, it doesn't seem to recognize the `gcc_toolchain` at all - the patch for this also need some revision.

Once these issues are resolved we should be closer to a successful build.

jeroen commented:
FWIW I was able to cross compile libv8 11.9 to arm64 without patching using the following configuration: https://github.com/jeroen/build-v8-static/blob/master/debian-11-arm/Dockerfile

danielnachun committed:
use native runners

danielnachun committed:
MNT: Re-rendered with conda-smithy 3.59.0 and conda-forge-pinning 2026.04.06.15.26.00

Other tools:
- conda-build 26.3.0
- rattler-build 0.61.4
- rattler-build-conda-compat 1.4.11

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I was trying to look for recipes to lint for you, but it appears we have a merge conflict. Please try to merge or rebase with the base branch to resolve this conflict.

Please ping the 'conda-forge/core' team (using the `@` notation in a comment) if you believe this is a bug.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found some lint.

Here's what I've got...


For **recipe/meta.yaml**:

 * ❌ Selectors are suggested to take a ``<two spaces>#<one space>[<expression>]`` form. See lines [61]
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).
 * ❌ When defining a source/url please add a sha256, sha1 or md5 checksum (sha256 preferably).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/24067329688](https://github.com/conda-forge/conda-forge-webservices/actions/runs/24067329688). Examine the logs at this URL for more detail.</sub>

danielnachun committed:
fix checksum
